src/lib/relay/createEnvironment.ts

```diff
@@ -1,8 +1,32 @@
-import { Environment, Network, RecordSource, Store } from "relay-runtime"
-import { fetchQuery } from "./fetchQuery"
+import { NativeModules } from "react-native"
+import { errorMiddleware, loggerMiddleware, RelayNetworkLayer, urlMiddleware } from "react-relay-network-modern/node8"
```

Comment:
I don't know if it matters, but we're on node 10 instead of 8.

Reply:
It doesn't matter here. This version makes use of new features that became available in node 8 (i.e. native async/await)